lib/services/irc.rb

```diff
@@ -137,6 +137,31 @@ def irc_puts(command, debug_command=command)
     writable_irc.puts command
   end
 
+  def irc_realname
+    repo_owner = payload["repository"]["owner"]["name"]
+    repo_name = payload["repository"]["name"]
+
+    "GitHub IRCBot - #{repo_owner}" + (repo_public? ? "/#{repo_name}" : "")
+  end
+
+  def repo_public?
```

Comment:
I wasn't certain. I couldn't find any other service doing a similar check, and the test payloads did not seem to contain them. 

However, I wrote a patch to verify and my tests did not fail. And it seems to function properly, I've added the commit to the PR.